docs/_documentations/remotedeploy-combo.md

```diff
@@ -10,59 +10,64 @@ parent: installoncloud
 order: 2
 ---
 
-## Installing a Codewind remote deployment
+# Installing a Codewind remote deployment all-in-one
 
-Codewind includes a CLI to simplify the installation process. You will find `cwctl` in your HOME directory under the path ~/.codewind/{version}
+Codewind includes a CLI to simplify the installation process. You can find the `cwctl` CLI in your HOME directory under the path `~/.codewind/{version}`.
 
-The command for installing an "all-in" deployment with a new Keycloak and new Codewind service is :
+The command for installing an all-in-one deployment with a new Keycloak and a new Codewind service is:
 
 `cwctl --insecure install remote` 
 
-and requires various flags to specify where and what to install.
+This command requires various flags to specify where and what to install.
 
-### Deploying Codewind with CWCTL
+## Deploying Codewind with CWCTL
 
-1.  Open a new terminal window on your local workstation
-2.  Navigate to your home directory and to the Codewind cli:
+1.  Open a new terminal window on your local workstation.
+2.  Navigate to your home directory, and then to the Codewind CLI:
 
 ```
 cd ~/.codewind/0.8.0
 ```
 
-Ensure you are logged into your Kubernetes or Openshift cluster
+Ensure that you are logged into your Kubernetes or Openshift cluster:
 
 ```
 $ kubectl get namespaces
 ```
 
-If the command is successful you should see a list of current namespaces.  If not,  ensure you are logged into your Kubernetes or Openshift cluster.
+If the command is successful, you see a list of current namespaces. If not, ensure that you are logged into your Kubernetes or Openshift cluster.
 
-### Determine your Cloud ingress domain
+## Determine your Cloud ingress domain
 
-The cli command requires an ingress domain,  you can find yours based on any of the existing routes:
+The CLI command requires an ingress domain. You can find your ingress domain based on any of the existing routes:
 
 ```
 oc get routes -n default
 NAME               HOST/PORT                                                                                                          PATH      SERVICES           PORT               TERMINATION   WILDCARD
 registry-console   registry-console-default.mycluster-12345-7674b4bd9abbdeea5be228236d5275c9-0001.eu-gb.containers.appdomain.cloud             registry-console   registry-console   passthrough   None
 ```
 
-In the above example the ingress domain needed is:
+In the above example the ingress domain is:
```

Comment:
Remove "above" because of accessibility reasons. Perhaps, "The preceeding example shows the following ingress domain:"